### ISSUE TITLE
Include bb addresses in cfg output

### ISSUE THREAD

**Issue by daverigby** [Kazhuu/asm2cfg#46]
It would be useful to include the addresses of the basic blocks in the cfg output - for large functions in particular it is hard to map code to the PDF CFG without knowing which the addresses the lines came from.

**Comment by Kazhuu** [comment]
Yeah this is nice addition indeed. If someone is willing to contribute a PR about this improvement it would be nice. At the moment though I'm waiting for the bigger refactoring PR https://github.com/Kazhuu/asm2cfg/pull/45 to get ready so I can help the author to work on it and minimize the differences between two branches :+1:
